Packs/Devo/ReleaseNotes/1_2_8.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Devo v2
+- Updated the Docker image to: *demisto/devo:1.0.0.74226*.
```

Comment:
Please add a sentence about fixing the bug.

Reply:
Added.